pkg/server/test/property_values_for_time_range_test.go

```diff
@@ -156,7 +156,7 @@ func Test_propertyValueForTimeRange_1h_data_for_time_range(t *testing.T) {
 			{
 				RefID:         "A",
 				QueryType:     models.QueryTypePropertyAggregate,
-				TimeRange:     backend.TimeRange{From: testdata.OneDay, To: testdata.Now},
+				TimeRange:     backend.TimeRange{From: testdata.FifteenDays, To: testdata.Now},
```

Comment:
Is there a reason we replaced OneDay with FifteenDays here?

Reply:
Yes - since there was no 15m option previously, it would jump the the hourly level aggregation much sooner, since the logic is to select a resolution based on how many data points can fit in a single page of results. 15 times more data can be fit into a request with 15m aggregation, compared to 1m aggregation